opensrp-app/src/test/java/org/smartregister/presenter/ListPresenterTest.java

```diff
@@ -54,6 +54,15 @@ public void testOnFetchRequestError() {
         Mockito.verify(view).setLoadingState(false);
     }
 
+    @Test
+    public void testOnFetchRequestErrorWithException() {
```

Comment:
Use more descriptive name for test cases
`test[MethodName][ExpectedResult][Parameters]`